apps/etterlatte-saksbehandling-ui/client/src/components/person/aktivitet/Aktivitet.tsx

```diff
@@ -0,0 +1,139 @@
+                        </Table.Row>
+                      </Table.Header>
+                      <Table.Body>
+                        {!!vurdering.aktivitet?.length ? (
+                          <>
+                            {vurdering.aktivitet.map((aktivitet) => {
+                              return (
+                                <Table.Row key={aktivitet.id}>
+                                  <Table.DataCell>
+                                    {tekstAktivitetspliktVurderingType[aktivitet.aktivitetsgrad]}
+                                  </Table.DataCell>
+                                  <Table.DataCell>{aktivitet.fom ? formaterDato(aktivitet.fom) : '-'}</Table.DataCell>
+                                  <Table.DataCell>{aktivitet.tom ? formaterDato(aktivitet.tom) : '-'}</Table.DataCell>
+                                  <Table.DataCell>{aktivitet.beskrivelse ? aktivitet.beskrivelse : '-'}</Table.DataCell>
+                                  <Table.DataCell>
+                                    {aktivitet.endret && formaterDatoMedFallback(aktivitet.endret.tidspunkt, '-')}
+                                  </Table.DataCell>
+                                  <Table.DataCell>
+                                    {aktivitet.endret.ident ? aktivitet.endret.ident : '-'}
+                                  </Table.DataCell>
+                                </Table.Row>
+                              )
+                            })}
+                          </>
+                        ) : (
+                          <Table.Row>
+                            <Table.DataCell colSpan={6}>Finner ingen aktivitetsgrad</Table.DataCell>
+                          </Table.Row>
+                        )}
+                      </Table.Body>
+                    </Table>
+                  </>
+                </VStack>
+                <VStack gap="4">
+                  <Heading size="xsmall">Unntak</Heading>
+                  <>
+                    <Table size="small">
+                      <Table.Header>
+                        <Table.Row>
+                          <Table.HeaderCell scope="col">Unntak</Table.HeaderCell>
+                          <Table.HeaderCell scope="col">Fra og med</Table.HeaderCell>
+                          <Table.HeaderCell scope="col">Til og med</Table.HeaderCell>
+                          <Table.HeaderCell scope="col">Beskrivelse</Table.HeaderCell>
+                          <Table.HeaderCell scope="col">Sist endret</Table.HeaderCell>
+                          <Table.HeaderCell scope="col">Saksbehandler</Table.HeaderCell>
+                        </Table.Row>
+                      </Table.Header>
+                      <Table.Body>
+                        {!!vurdering.unntak?.length ? (
+                          <>
+                            {vurdering.unntak.map((unntak) => {
+                              return (
```

Comment:
Trenger ikke return

Reply:
Fikset på map for både unntak og aktivitet